admin/workspace-management/process-logging.md

```diff
@@ -65,10 +65,25 @@ To view the process logs from a specific user or workspace, you can use your
 cloud provider's log viewer, or you can use `kubectl` to print the logs:
 
 ```bash
-kubectl logs \
-  -l "com.coder.username=zac" \        # Filter by the user "zac"
-  -l "com.coder.workspace.name=code" \ # Filter by the workspace "code"
-  -c exectrace                         # Only show logs from the sidecar
+kubectl logs /
+--selector="com.coder.username=admin" /         # Filter by the user "admin"
+--selector="com.coder.workspace.name=main" /    # Filter by the workspace "main"
+-c workspace                                    # Only show logs from the workspace
```

Comment:
`exectrace` not `workspace`, and you'll have to update the example logs below. You can't get these on our GKE clusters because GKE doesn't support exectrace, you'll have to enable the feature on an Ubuntu 20.04 cluster on EKS.